### PR TITLE
Issue 508/defining measures bug

### PR DESCRIPTION
**Pull request by mikeAdamss**
bug fix for: https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/508

### CODE REVIEW

File: csvcubed/csvcubed/models/cube/qb/validationerrors.py [hunk 1]
```diff
@@ -308,3 +308,18 @@ class BothMeasureTypesDefinedError(IncompatibleComponentsError):
     @classmethod
     def get_error_url(cls) -> str:
         return 'http://purl.org/csv-cubed/err/both-meas-typ-def'
+
+
+@dataclass
+class EmptyQbMultiMeasureDimensionError(SpecificValidationError):
```
[the remark this answers is not in the document]
Comment (mikeAdamss) [reply csvcubed/csvcubed/models/cube/qb/validationerrors.py:314]: reading that file is the worst thing that's happened to me today 😬

File: src/csvcubed/models/cube/qb/validationerrors.py [hunk 2]
```diff
@@ -308,3 +308,18 @@ class BothMeasureTypesDefinedError(IncompatibleComponentsError):
...
+@dataclass
+class EmptyQbMultiMeasureDimensionError(SpecificValidationError):
+    """
+    An error for when the user has a `QbMultiMeasureDimension` but its measure
+    field is an empty list.
+    """
+
+    @classmethod
+    def get_error_url(cls) -> str:
+        return 'http://purl.org/csv-cubed/err/empty-multi-meas-dimension'
```
Comment (robons) [inline src/csvcubed/models/cube/qb/validationerrors.py:322]: The documentation pages corresponding to these URIs need to be part of this PR too.

This is the location where they should be created:

https://github.com/GSS-Cogs/csvcubed/tree/main/external-docs/docs/guides/errors/build-command-errors

Once they've been set up, we can create the PURL url mappings.

**Review by robons — CHANGES_REQUESTED** [review]
Looks good, but make sure you add the help documentation in for those new errors you're put in.

**Review by robons — APPROVED** [review]
Don't forget those PURL URLs are made before the next release.

**Pull request merged**